### PR TITLE
Add docs for the java main sti image.

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

**Comment by adellape** [comment]
Thanks, @iocanel. Can you add the new topic to the `_build_cfg.yml` file, per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/contributing.adoc#how-it-all-comes-together?

File: _build_cfg.yml [hunk 1]
```diff
@@ -227,6 +227,8 @@ Topics:
         File: php
       - Name: Python
         File: python
+      - Name: Java
+      - File: java
```
Comment (adellape) [inline _build_cfg.yml]: You can remove the `-` in front of `File` here, then the build should pass.

**Comment by adellape** [comment]
cc @tpoitras in case you have any feedback at this stage.

**Comment by tpoitras** [comment]
@iocanel -- are there any other relevant environment variables for this image?

(thanks @adellape )

**Comment by iocanel** [comment]
No other env vars are available. Just JAVA_MAIN, MAVEN_ARGS and OUTPUT_DIR (which are already described).

**Comment by adellape** [comment]
Thanks, @iocanel; merging. Relevant docs card for any follow-up: https://trello.com/c/LnGCcW7G

**Pull request merged**